slicedimage/backends/_caching.py

```diff
@@ -9,6 +9,7 @@
 from ._base import Backend
 
 SIZE_LIMIT = 5e9
+CACHE_VERSION = "v0"
```

Comment:
will we ever increment this?

Reply:
Next time we have a cache corruption bug.